dataactbroker/handlers/fileHandler.py

```diff
@@ -411,48 +449,62 @@ def uploadFile(self):
             # Unexpected exception, this is a 500 server error
             return JsonResponse.error(e,StatusCode.INTERNAL_ERROR)
 
-    def generateD1File(self):
-        """ Initiates the generation of D1 """
-        jobDb = self.interfaces.jobDb
-        requestDict = RequestDictionary(self.request)
+    def startGenerationJob(self, submission_id, file_type):
+        """ Initiates the generation of D1
```

Comment:
Thanks for updating the docstring--does this part specific to the D1 file need to be changed too?